src/components/MessageTab.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from "react";
+
+export const MessageTab = () => {
```

Comment:
Looking at our [mock API docs](https://github.com/tripleten-externships/communitee-mock-api), we can see that the endpoint we'll end up using here is to fetch all of our messages. As such, it's reasonable to think we'll pass in the messages (and unread count) from a component higher up in the hierarchy. So let's bring the unread count as a prop here. This will also make it such that we can make it a prop for storybook and adjust/test it there.